containers/unit-tests/start

```diff
@@ -3,31 +3,17 @@ set -eu
 
 cd "${0%/*}/../.."
 image=ghcr.io/cockpit-project/unit-tests:latest
-flags=''
+cmd=''
 
 while test $# -gt 0; do
   case "$1" in
-    --docker) docker="docker";;
```

Comment:
Feel free to eliminate this script entirely, as the commit message suggests. I doubt that anyone is "familiar" with that API, hardly noone except perhaps you actually runs this manually. There is already a gazillion scripts around this, and it's really hard to understand where the functionality actually lives -- so anything which can be removed is a little win.

Reply:
Ya.  Sounds good.

Where would you like to see the new script live, btw?  `tools/run-container` seemed kinda ... I don't know.  Other ideas I had:
 - in `containers/unit-tests`
 - in `containers` itself
 - other ideas?

Reply:
containers/unit-tests/run or /start feels about right to me -- let's not generalize this until we actually need it (if at all), and we may be able to simplify it while it stays unit-tests specific?